index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   rules: requireIndex(path.join(__dirname, '/custom-rules')),
   configs: {
     base: require('./config/base'),
```

Comment:
I think we might not even need to expose `config/base` anymore. That was exposed earlier because prettier was optional, but now that it's not, I think we don't need to expose it anymore!